src/microbe_stage/MicrobeColony.cs

```diff
@@ -71,6 +71,24 @@ public void RemoveFromColony(Microbe microbe)
         if (State == Microbe.MicrobeState.Unbinding)
             State = Microbe.MicrobeState.Normal;
 
+        var rotation = microbe.RotationInsideColony();
+        var offset = microbe.GetOffsetRelativeToMaster();
+
+        // Apply the rotation
```

Comment:
Isn't this where the "magic" of this PR happens. I think this comment needs to be explained more in detail *or* this whole if contents should be extracted as a named function to give this code the proper "label" to explain why it exists.